app/common/logger.py

```diff
@@ -0,0 +1,26 @@
+import logging
+from telegram import Update
+from telegram.ext import (CallbackContext)
+from config import LOGGING_CHAT_ID
+
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+
+class Logger:
+
+    def __init__(self, name):
+        self._logger = logging.getLogger(name)
+        self._logger.setLevel(logging.INFO)
+
+    def info(self, message: str):
+        self._logger.info(message)
+
+    def error(self, message: str):
+        self._logger.warning(message)
+
+    def admin(self, message: str):
+        """Logs the message to the logging chat"""
+        context.bot.send_message(chat_id=LOGGING_CHAT_ID,
+                             text=message, disable_web_page_preview=True)
+
+
```

Comment:
```suggestion
import logging

class Logger(logging.Logger):

    def __init__(self, name: str, level=logging.NOTSET: int):
        super().__init__(name, level)
       
    def admin(self, message: str) -> None: # and here an example of custom method ?
        ...
```
Could be this not better?
and then if we want to override some methods, we could do that instead of rewriting the new class? 🤔

Reply:
I don't have an idea about how to test the admin log.